client/package-lock.json

```diff
@@ -0,0 +1,17420 @@
+{
```

Comment:
This file's been removed after we switched to yarn, can you reinstall the packages you added using yarn and delete this file? Probably just a merge remnant.

`yarn workspace client add --save react-icons react-share`